web3/_utils/method_formatters.py

```diff
@@ -485,8 +485,13 @@ def raise_solidity_error_on_revert(response: RPCResponse) -> RPCResponse:
     if not isinstance(response['error'], dict):
         raise ValueError('Error expected to be a dict')
 
-    # Parity/OpenEthereum case:
     data = response['error'].get('data', '')
+
+    # Ganache case:
+    if isinstance(data, dict) and response['error'].get('message'):
+        raise SolidityError(response['error']['message'])
```

Comment:
I think it would be a good idea to standardize this response with the Geth response. Do you mind adding an `execution reverted` to the beginning of this error message? So:

`raise SolidityError(f"execution reverted: {response['error']['message']}")`

Reply:
thanks for the review, fixed 👌